app/api/student.py

```diff
@@ -1,40 +1,50 @@
+import logging
+import typing as tp
+from db.db import Database, BriefQueue, Queue, ComparatorType
 
 
 class Student:
 
-    def __init__(self, student_tg_name):
-        self.student_tg_name = student_tg_name
-        self.groups = []
-        self.review_queues = []
+    def __init__(self, student_tg_id: int, database: Database) -> None:
+        self.student_tg_id = student_tg_id
+        self.database = database
+        self.is_subscribed_review_queue_enroll_start_notifications = True
```

Comment:
Probably subscription should be not in the Student constructor, because it could be called multiple times for a student (not only in case of a new student creation)

Reply:
For now we decided to subscribe all students to notifications by default, so I commented out [this](https://github.com/six-nine/edu_queue/blob/student_api_interface/app/api/student.py#L44) methods

Reply:
OK, if there is no additional logics on it, let's leave it as is.
(But if you will make additional changes, maybe worth it to comment this line too)